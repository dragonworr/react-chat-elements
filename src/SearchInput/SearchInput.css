.rce-container-search-input {
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: space-between;
    overflow: hidden;
    width: 100%;
    box-sizing: border-box;
    background: #fff;
    padding-right: 5px;
}