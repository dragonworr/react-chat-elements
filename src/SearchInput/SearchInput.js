import React, { Component } from 'react';
import './SearchInput.css';
import {
	Button,
	Input,
} from '../../src';

const classNames = require('classnames');

export class SearchInput extends Component {
	render() {
		return (
			<div className="rce-container-search-input">
				{/* <input
					type={this.props.type}
					className="rce-search-input"
					placeholder={this.props.placeholder}
					value={this.state.value} /> */}
				<Input
					placeholder="Mesajınızı buraya yazınız." defaultValue="" />
				<Button
					type="default" text="Search" />
			</div>
		);
	}
}

SearchInput.defaultProps = {
	type: 'text',
	placeholder: '',
	value: ''
};

export default SearchInput;