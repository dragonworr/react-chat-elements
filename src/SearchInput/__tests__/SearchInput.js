import React, { Component } from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchInput from '../SearchInput';

describe('SearchInput component', () => {
  it('should render without issues', () => {
    const component = shallow(<SearchInput />);
    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});