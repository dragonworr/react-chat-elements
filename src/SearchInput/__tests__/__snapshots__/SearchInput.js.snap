// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput component should render without issues 1`] = `
<div
  className="rce-container-search-input"
>
  <Input
    defaultValue=""
    onChange=""
    placeholder="Mesajınızı buraya yazınız."
    type="text"
  />
  <Button
    backgroundColor="#3979aa"
    color="white"
    disabled={false}
    icon={null}
    text="Search"
    type="default"
  />
</div>
`;
